readme.md

```diff
@@ -368,6 +368,28 @@ import bourne from '@hapijs/bourne';
 })();
 ```
 
+##### fetch
+
+Type: `Function`\
+Default: `fetch`
+
+User-defined `fetch` function.
+
+Use-cases:
+1. Use custom `fetch` implementations like [`isomorphic-unfetch`](https://www.npmjs.com/package/isomorphic-unfetch).
+2. Use the `fetch` wrapper function provided by some frameworks that use server-side rendering (SSR).
+
+```js
+import ky from 'ky';
+import fetch from 'isomorphic-unfetch';
+
+(async () => {
+	const parsed = await ky('https://example.com', {
+		fetch: fetch
```

Comment:
Since the key and value are the same, you can use the shorthand syntax and just write `fetch` instead of `fetch: fetch`.